packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/createproject.md

```diff
@@ -0,0 +1,3 @@
+Create a new project using the [Spring Initializer](https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=11&groupId=com.example&artifactId=demo&name=demo&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.demo&dependencies=web,okta). Using this link automatically selects the **Web** and **Okta** dependencies.
+
+> **Note**: This guide uses Spring Boot 2.6.3.
```

Comment:
```suggestion
```

cc @mraible 

This is going to get out of date fast, the above link does NOT include the version. IMHO, we should either remove the note, or change it to to reference the minor version, something like:

> > **Note**: This guide was tested with Spring Boot 2.6.

Reply:
I like "this guide was tested with".

Reply:
Good call. I've updated this in the api and web spring QSes.